packages/docs/celo-codebase/protocol/identity/accounts.md

```diff
@@ -0,0 +1,75 @@
+# Key management
+
+There is a pretty funny [tweet](https://twitter.com/LeaKissner/status/1198595109756887040) which goes something like this:
+
+> Crypto is a tool for turning a whole swathe of problems into key management problems. Key management problems are way harder than (virtually all) cryptographers think.
+
+To help with this, we have various mechanisms to reduce the impact of the loss or compromise of keys, mainly through our `Accounts` smart contract. On it, accounts can authorize other keys to perform certain actions on behalf of the account. Keys that need to be accessed frequently (e.g. for signing blocks) are at greater risk of being compromised, and thus have more limited permissions, while keys that need to be accessed infrequently (e.g. for locking Celo Gold) are less onerous to store securely, and thus have more expansive permissions. Below is a summary of the various keys that are used in the Celo network, and a description of their permissions.
+
+| Name of the key        | Purpose                                                                                                                                                                                                                                                               |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Account key            | This is the key with the highest level of permissions, and is thus the most sensitive. It can be used to lock and unlock Celo Gold, and authorize vote, validator, and attestation keys. Note that the account key also has all of the permissions of the other keys. |
```

Comment:
The last sentence isn't entirely true. Once a signer is authorized, the account key can no longer serve that function. (i.e. if a validator key is authorized, the account key can no longer sign consensus messages)
```suggestion
| Account key            | This key has the highest level of permissions, and is thus the most sensitive. It can be used to lock and unlock Celo Gold, and authorize vote, validator, and attestation keys. |
```